index.php

```diff
@@ -12,7 +17,32 @@
             <h1> Flock Information Database Home </h1>
         </div>        
         <div class="content">
-            <h2> Stick a login form here </h2>
+            
+            <center>
+                <h3>User Login</h3>
+                <hr />
+                <form method="post" action="redirectPostLogin.php">
+                    <table border="0">
+                        <tr>
+                            <td><b>Username</b></td>
```

Comment:
I think using a label tag would be more suitable here since that's what they're for, labelling input tags.
This line would be 
```html
<td><label for="username"><b>Username</b></label></td>
```